### ISSUE TITLE
[BUG] val_strategy='bootstrap' does not create .json file for the bootstrapped split to be referenced later

### ISSUE THREAD

**Issue by siddhir** [jamesdolezal/slideflow#301]
## Description
.json file is not created with splits = /file/to/bootstrap_split.json

## To Reproduce
boot_train,boot_val = train_dataset.split(
    val_strategy='bootstrap',
    val_fraction=0.2,
    labels='progressor',
    splits='/mnt/data/PROJECTS/HNSC_PREMAL/splits/bootstrap_split.json'
)

## Expected behavior
k-fold splits are saved appropriately but boostrapped splits are not

**Comment by jamesdolezal** [comment]
This is actually intended behavior, as bootstrap validation involves random partitioning of data into training/validation, and these random partitions are never reused.
